pkg/scaffold/project.go

```diff
@@ -122,8 +127,18 @@ func (p *V1Project) Scaffold() error {
 	imgName := "controller:latest"
 
 	s = &Scaffold{}
+
+	universe, err = model.NewUniverse(
+		model.WithProject(&p.Project.ProjectFile),
+		model.WithBoilerplateFrom(p.Boilerplate.Path),
+		// No resource at project initialization
+	)
+	if err != nil {
+		return fmt.Errorf("error creating scaffolding universe: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("error creating int project: %v", err)
```

Reply:
Will probably go for `"error initializing project: %v"`